modules/core/elpaca.el

```diff
@@ -0,0 +1,53 @@
+;;; core/elpaca --- DESCRIPTION -*- no-byte-compile: t; lexical-binding: t; -*-
+
+;; BEGIN ELPACA SNIPPET: https://github.com/progfolio/elpaca?tab=readme-ov-file#installer
```

Comment:
As you can see from this, this is just a snippet that comes directly from that link.

Reply:
Made some tweaks to resolve some issues I had with `elpaca-core-date`